config/kernel/linux-rockchip-rk3588-legacy.config

```diff
@@ -2404,7 +2404,7 @@ CONFIG_NET_TEAM=m
 # CONFIG_NET_TEAM_MODE_RANDOM is not set
 # CONFIG_NET_TEAM_MODE_ACTIVEBACKUP is not set
 # CONFIG_NET_TEAM_MODE_LOADBALANCE is not set
-CONFIG_MACVLAN=m
+CONFIG_MACVLAN=y
```

Comment:
I can confirm that adding `macvlan` to `/etc/modules-load.d/modules.conf` can auto load this module at boot.

Reply:
ok this is better.